blog/02-03-2023-basics-of-blockchain/basics-of-blockchain.md

```diff
@@ -0,0 +1,197 @@
+The Department of Homeland Security is researching blockchain approaches to data security.
+
+So one of the major sectors which can take the advantage of the technology is the government and banking sectors, and I am in hope that these sectors take the blockchain seriously because these sectors have the maximum possibility to make the transition from the web2 tech stack to web3 tech stack smoothly for the daily users, the transition has to be so smooth that they should be even aware of that the backend part has been updated or changed because we all know the majority of daily users are not worried about the technology the app or product is using, they just want to have a smoother hassle-free experience.
+
+After knowing these things let me tell you some of the other use cases where the blockchain is used like the below-mentioned things
+
+1. NFT
+NFT stands for Non-Fungible Token, it's much more than just ape pictures and a million-dollar rock picture. In short, it is a digital asset that represents the ownership of a unique item, NFT means that you cannot exchange that for the same item which makes them unique. They are stored on the Blockchain.
+Now many of us may wonder how the prices of NFT are decided, why one NFT is worth millions of dollars while the other just gets free by minting.
+In short the more rare and popular it is, the more price it has, if the NFT has its fanbase kind of thing, it will also affect the prices by a big factor.
+A longer answer, it also has the same policy as any other product in the market, its price is determined by the demand and supply of that NFT, and the reputation and popularity of not only the art but of the artist also matter. 
+In other tan art pieces the NFTs can also be used to show the ownership of the virtual digital assets, any in-game accessories, or any weapon which can be used in any game if you have that NFT on your address, and seeing the METAverse getting all the attention more and more nowadays, this thing will be utilized in many unique and many ways then we can think of today.
+Building towards that future.
+
+There are different types of NFT, and some of them I would like to discuss are
+- Art NFTS-As the name suggests they represent the ownership of unique art
+- Music NFTs- they represent the ownership of unique music 
+And such like that there are other NFTs such as Utility NFTs, Ticket NFTs, Video NFTs, Collectible NFTs
+
+There are two general types of NFT Static NFTs and Dynamic NFTs
+
+Static NFT- Is a digital asset that won’t be able to change or modified in any way, Once it is created you won’t be able to change that and this makes it unique
+
+Dynamic NFT-Whereas in a Dynamic NFT can be modified in the future, such as a game or an environment.
+
+Static NFTs are more common to see around.
+
+After reading these portions if you want to be the owner of a cool NFT, you can check that out on these amazing marketplaces- OpenSea, Rarible, SuperRare, and KnownOrigin
+
+2. DAO
```

Comment:
Please add extra line after here.
Note, there are lot of changes that requires adding new line, so I suggest using Prettier for this. That will format the markdown properly.

Thanks